proxy/static/index.html

```diff
@@ -1,97 +1,117 @@
 <!doctype html>
-<html lang="en">
```

Comment:
Nitpick: W3's HTML validator says we should use a `lang` attribute here. I don't know enough about HTML to know if we should care.